Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1904,6 +1904,110 @@ theorem lt_iff_sub_pos {x y : PGame} : x < y ↔ 0 < y - x :=
       ⟩
 #align pgame.lt_iff_sub_pos SetTheory.PGame.lt_iff_sub_pos
 
+/-! ### Inserting an option -/
+
+/-- The pregame constructed by inserting x' as a new left option into x. -/
```

Comment:
```suggestion
/-- The pregame constructed by inserting `x'` as a new left option into `x`. -/
```
since these are variable names, and this renders as markdown in vscode / the web docs. Same throughout your docstrings.